html/css/widgets-styles.css

```diff
@@ -0,0 +1,17 @@
+.foreign {
+    text-align: left ;
+}
+.insideforeign {
+    display: inline-block;
+    width: 300px;
+}
+
+.contentEditable.growToLeft{
```

Comment:
Not sure yet if these class names have been defined by you or some external library / api. If they have been defined by you, better use hyphens to separate words instead of camel case (that is tthe convention)